v19.1/gssapi_authentication.md

```diff
@@ -0,0 +1,106 @@
+---
+title: GSSAPI Authentication (Enterprise)
+summary: Learn about the GSSAPI authentication features for secure CockroachDB clusters.
+toc: true
+---
+
+CockroachDB supports the Generic Security Services API (GSSAPI) with Kerberos authentication as an [Enterprise feature](enterprise-licensing.html).
+
+## Configuring KDC
+
+To use Kerberos authentication with CockroachDB, configure a Kerberos service principal name (SPN) for CockroachDB and generate a valid keytab file with the following specifications:
+
+- Set the SPN to `postgres`.
```

Comment:
@roland Do you happen to know why this is required? I don't. When I changed our docker compose test to use `cockroach/` instead of `postgres/` it fails. We don't hard code the word `postgres` in our GSS code. So I *think* (?) that it must be coming from the psql client. If that's the case, then this step isn't actually required, and could instead be specifiable by whatever client they are using. This currently the only area where I'm not sure what's going on.

Reply:
It's specified by the client driver using the [`krbsrvname`](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-KRBSRVNAME) connection parameter. I believe any value should work as long as you specify the same value in the SPN and the connection string/URL. I wouldn't mind if we made it explicit that you can choose whatever you want and that `postgres` is just the default used by libpq, but it's not extremely important to me.

Reply:
Thanks. We should probably change this to note that if you are testing with psql, it should be postgres, otherwise it can be whatever you configure.